cpp/test/Ice/operations/Collocated.cpp

```diff
@@ -12,22 +14,47 @@ class Collocated : public Test::TestHelper
     void run(int, char**) override;
 };
 
+namespace
+{
+
+    void testCollocatedIPv6Invocation()
+    {
+        cout << "testing collocated invocation with normalized IPv6 address... " << flush;
+        Ice::CommunicatorHolder communicator = Ice::initialize();
+        communicator->getProperties()->setProperty("TestAdapter.Endpoints", "tcp -h \"0:0:0:0:0:0:0:1\" -p 10000");
```

Comment:
Should we use an ephemeral port or a test helper port to avoid consuming port 10,000 for a little while?

Reply:
The adapter is not activated, does it still consume the endpoint?

Reply:
You are right, better use a helper port.